tests/test_filtering.py

```diff
@@ -244,3 +294,46 @@ def test_error_invalid_field(client):
         'detail': 'Not a valid integer.',
         'source': {'parameter': 'size_min'},
     }])
+
+
+def test_error_column_filter_required_missing(client):
+    response = client.get('/widgets_size_required')
+    assert_response(response, 400, [{
+        'code': 'invalid_filter.missing',
+        'source': {'parameter': 'size'},
+    }])
+
+
+def test_error_model_filter_required_missing(client):
+    response = client.get('/widgets_color_custom')
+    assert_response(response, 400, [{
+        'code': 'invalid_filter',
+        'detail': 'Missing data for required field.',
```

Comment:
This is inconsistent with the above. A bit annoying. I think I want to move the parts of the code after the dots to an `extra` or a `meta.extra` field.